### PR TITLE
hud10837/set initial map area

### PR DESCRIPTION
**Pull request by hud10837**
This is the kotlin version of Set Initial Map Area. @TADraeseke can you take a look when you get the chance?

### CODE REVIEW

File: kotlin/set-initial-map-area/README.metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        "set-initial-map-area.png"
+    ],
+    "keywords": [
+        "extent",
+        "zoom"
+    ],
+    "redirect_from": "/android/latest/sample-code/set-initial-map-area.htm",
+    "relevant_apis": [
+      "ArcGISMap",
+      "Envelope"
```
Comment (TADraeseke) [inline kotlin/set-initial-map-area/README.metadata.json:15]:
```suggestion
      "Envelope",
```

File: kotlin/set-initial-map-area/src/main/java/com/esri/arcgisruntime/sample/setinitialmaparea/MainActivity.kt [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import com.esri.arcgisruntime.mapping.Basemap
+import com.esri.arcgisruntime.mapping.Viewpoint
+import kotlinx.android.synthetic.main.activity_main.*
+
+class MainActivity : AppCompatActivity() {
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_main)
+    // create a map with the basemap
```
Comment (TADraeseke) [inline kotlin/set-initial-map-area/src/main/java/com/esri/arcgisruntime/sample/setinitialmaparea/MainActivity.kt:32]:
```suggestion
    // create a map with a topographic basemap
```

File: kotlin/set-initial-map-area/README.metadata.json (hunk 1 above)
Comment (hud10837) [reply kotlin/set-initial-map-area/README.metadata.json:15]: good catch!

**Comment by hud10837** [comment]
@TADraeseke Thanks! I've fixed those now

**Comment by hud10837** [comment]
Thanks! Merging now

**Pull request merged**